src/config.js

```diff
@@ -11,6 +11,9 @@ const DISCORD_SERVER = {
   scheduledDiscordEventsEnabled:
     process.env.SCHEDULED_DISCORD_EVENTS_ENABLED === 'true',
 };
+const CHANNEL_NAME = {
```

Comment:
add a more descriptive name please, for example something related to the feature "LISTEN_NEW_EVENTS_REPORT_CHANNEL" or similar